HoneyBadgers.WebApp/Controllers/ReportController.cs

```diff
@@ -40,6 +42,7 @@ public async Task<ActionResult<ReportGenreStatsModel>> Index()
 
 
         [HttpGet]
+        [Route("getreportdata")]
```

Comment:
ta nazwa chyba nie jestpotrzebna bo to jest dokładnie taka sama nazwa jak nazwa metody

Reply:
poprawiło mi czytelność :-) funkcjonalności nie poprawia ale i nie szkodzi